tldw_Server_API/app/core/DB_Management/guardian_db_resolver.py

```
@@ -0,0 +1,25 @@
+
+import hashlib
+
+from tldw_Server_API.app.core.DB_Management.Guardian_DB import GuardianDB
+from tldw_Server_API.app.core.DB_Management.db_path_utils import DatabasePaths
+
+
+def coerce_guardian_storage_user_id(raw_user_id: object) -> int:
+    """Normalize user IDs to the integer storage key used for guardian DB paths."""
+    try:
+        return int(raw_user_id)
+    except Exception:
```

Ruff (0.15.9)

[warning] Do not catch blind exception: `Exception`

(BLE001)